Bottom: 3fc80cc3a3a099f1ed2cd784b31882c8238a8a52
Top:    d8d223bf77ea14e3973477682d504ca9b1eda126
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-30 14:26:49 -0500

Rearrange DEBIAN_FRONTEND envar in Dockerfiles


---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index c85bf86..05d87e0 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -12,9 +12,9 @@ MAINTAINER  Buildbot maintainers
 WORKDIR /buildbot
 
 # Install security updates and required packages
-RUN DEBIAN_FRONTEND=noninteractive \
-    apt-get update \
-&&  apt-get -y install -q \
+RUN apt-get update \
+&&  DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install -q \
        alien \
        build-essential \
        chrpath \
